NEWS

```diff
@@ -297,6 +331,45 @@ CHANGES WITH 248:
           by programs for detecting whether they were forked off by the service
           manager itself or are a process forked off further down the tree.
 
+        * The sd-device API gained three new calls sd_device_get_action() (for
+          determining the uevent add/remove/change/… action the device object
+          has been seen for), sd_device_get_seqno() (for determining the uevent
+          sequence number) and sd_device_new_from_stat_rdev() (for allocating a
+          new sd_device object from stat() data of a device node).
+
+        * For most tools the --no-legend= switch has been replaced by
+          --legend=no and --legend=yes, to force whether tables are shown with
+          headers/legends.
```

Comment:
"For most tools"?? If I remember correctly, `--legend` is only accepted by `systemctl`? @keszybz?